pysmt/test/test_bv_simplification.py

```diff
@@ -94,6 +95,316 @@ def all_special(self):
                 for op in self.special_operators:
                     self.check(op(l, n))
 
+    def test_bv_0_add(self):
+        x = Symbol("x", BVType(32))
+        f = BVAdd(BVZero(32), x)
+        self.assertEqual(f.simplify(), Symbol("x", BVType(32)))
+
```

Comment:
I would use directly x in the assert. It is more clear.

Reply:
Will change in the next PR.